plugins/plugin-chart-table/test/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "composite": false,
     "emitDeclarationOnly": false,
     "noEmit": true,
-    "rootDir": "."
+    "rootDir": "../../../"
```

Comment:
Is this intentional?

Reply:
Yes as I need to import a contextualized `enzyme` mount from root: https://github.com/ktmud/superset-ui/blob/8e744a77751fc8863522c8938974262e7e3463bd/plugins/plugin-chart-table/test/TableChart.test.tsx#L21